### PR TITLE
'Connect Wallet' button instead of empty account

### PR DESCRIPTION
**Pull request by nepoche**

### CODE REVIEW

**Comment by AhmedKorim** [comment]
@nepoche  ,The button for adding custom EVM RPC isn't working anymore

**Comment by nepoche** [comment]
Just attempted on this branch to add 'Harmony Testnet Shard 1' custom EVM RPC. It seemed to work fine for me.

**Comment by AhmedKorim** [comment]
This seems not to work for me
- Network switching throw MetaMask doesn’t do anything
- dApp connection never done just keeps showing connecting

File: packages/ui-components/src/Inputs/WalletSelect/ConnectWalletButton.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+          }
+          return {
+            ...walletConfig,
+            enabled: isDetected,
+            connected,
+            async endSession() {},
+            canEndSession: false,
+          };
+        })
+      );
```
Comment (AhmedKorim) [inline packages/ui-components/src/Inputs/WalletSelect/ConnectWalletButton.tsx:68]: This is used as well on the `WalletSelect.tsx` can we share logic between both places @nepoche

**Review by AhmedKorim — CHANGES_REQUESTED** [review]
Looks good need just a small refactor for the `configureSelectedWallets`

**Comment by drewstone** [comment]
@AhmedKorim is this ready to be merged or do you still have to review? If it's too stale can we close this?

**Comment by nepoche** [comment]
I would suggest merging this in because it has a bugfix regarding targeting of chains, or I can split this out in a separate PR: https://github.com/webb-tools/webb-dapp/pull/134/files#diff-c688cc440af23b5cb634e02d91c0efbc31ac3a7fe4dc0dc982ecf18417783af4R26

**Comment by AhmedKorim** [comment]
Looks good!

**Pull request merged**